src/main/java/hudson/plugins/analysis/util/model/AbstractAnnotation.java

```diff
@@ -350,7 +352,7 @@ public final String getFileName() {
     }
 
     @Override
-    public String getTempName(final AbstractBuild<?, ?> owner) {
+    public String getTempName(final Run<?, ?> owner) {
```

Comment:
Added at the end of the class:

```
/**
 * @deprecated use {@link #getTempName(Run)} instead
 */
@Deprecated
public String getTempName(final AbstractBuild<?, ?> owner) {
    return getTempName((Run<?, ?>) owner);
}
```

Not sure about the ```null``` parameters you mentioned. Why is this method different than others in this PR (with similar signature)?